src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -253,10 +253,18 @@ private void UpdateEnvironmentVariables(IDictionary<string, string> secrets)
         public override async Task RunAsync()
         {
             await PreRunConditions();
-            if (VerboseLogging.HasValue && VerboseLogging.Value)
+            var displayLogo = Environment.GetEnvironmentVariable(Constants.DisplayLogoVariable) == "1";
+            var isVerbose = VerboseLogging.HasValue && VerboseLogging.Value;
+            if (isVerbose || displayLogo)
             {
                 Utilities.PrintLogo();
             }
+
+            if (!isVerbose)
+            {
```

Comment:
good to add E2E tests.  If its ok, I can add tests and push to this branch?

Reply:
Yes. Thanks!